doc/tutorial/using_gpu_solution_1.py

```diff
@@ -101,171 +88,171 @@
+   ... (remaining 2 Apply instances account for 0.00%(0.00s) of the runtime)
+
 
 
 
 # 2.2 Profiling for GPU computations
 
 # In your terminal, type:
-$ CUDA_LAUNCH_BLOCKING=1 THEANO_FLAGS=profile=True,device=gpu python using_gpu_solution_1.py
+$ CUDA_LAUNCH_BLOCKING=1 THEANO_FLAGS=profile=True,device=cuda python using_gpu_solution_1.py
 
 # You'll see first the output of the script:
 Used the gpu
 target values for D
 prediction on D
 
-Results were produced using a GeForce GTX TITAN
+Results were produced using a GeForce GTX TITAN X
 
 # Profiling summary for all functions:
 
 Function profiling
 ==================
-  Message: Sum of all(3) printed profiles at exit excluding Scan op profile.
-  Time in 10002 calls to Function.__call__: 3.535239e+00s
-  Time in Function.fn.__call__: 3.420863e+00s (96.765%)
-  Time in thunks: 2.865905e+00s (81.067%)
-  Total compile time: 4.728150e-01s
-    Number of Apply nodes: 36
-    Theano Optimizer time: 4.283385e-01s
-       Theano validate time: 7.687330e-03s
-    Theano Linker time (includes C, CUDA code generation/compiling): 2.801418e-02s
-
+  Message: Sum of all(2) printed profiles at exit excluding Scan op profile.
+  Time in 10001 calls to Function.__call__: 4.181247e+00s
+  Time in Function.fn.__call__: 4.081113e+00s (97.605%)
+  Time in thunks: 3.915566e+00s (93.646%)
+  Total compile time: 9.256095e+00s
+    Number of Apply nodes: 21
+    Theano Optimizer time: 9.996419e-01s
+       Theano validate time: 6.523132e-03s
+    Theano Linker time (includes C, CUDA code generation/compiling): 8.239602e+00s
+       Import time 4.228115e-03s
+
+Time in all call to theano.grad() 3.286195e-02s
+Time since theano import 15.415s
 Class
 ---
 <% time> <sum %> <apply time> <time per call> <type> <#call> <#apply> <Class name>
-  45.7%    45.7%       1.308s       1.64e-05s     C    80001       9   theano.sandbox.cuda.basic_ops.GpuElemwise
-  17.2%    62.8%       0.492s       2.46e-05s     C    20002       4   theano.sandbox.cuda.blas.GpuGemv
-  15.1%    77.9%       0.433s       2.17e-05s     C    20001       3   theano.sandbox.cuda.basic_ops.GpuAlloc
-   8.2%    86.1%       0.234s       1.17e-05s     C    20002       4   theano.sandbox.cuda.basic_ops.HostFromGpu
-   7.2%    93.3%       0.207s       2.07e-05s     C    10000       1   theano.sandbox.cuda.basic_ops.GpuCAReduce
-   4.4%    97.7%       0.127s       1.27e-05s     C    10003       4   theano.sandbox.cuda.basic_ops.GpuFromHost
-   0.9%    98.6%       0.025s       8.23e-07s     C    30001       4   theano.sandbox.cuda.basic_ops.GpuDimShuffle
-   0.7%    99.3%       0.020s       9.88e-07s     C    20001       3   theano.tensor.elemwise.Elemwise
-   0.5%    99.8%       0.014s       7.18e-07s     C    20001       3   theano.compile.ops.Shape_i
-   0.2%   100.0%       0.006s       5.78e-07s     C    10000       1   theano.tensor.elemwise.DimShuffle
+  59.5%    59.5%       2.329s       1.16e-04s     C    20001       3   theano.sandbox.gpuarray.blas.GpuGemv
```

Comment:
Clearly something is wrong with GpuGemv here since it's 10 times slower on a faster GPU.